ams_flutter/lib/features/onboarding/presentation/pages/markAttendance.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:ams_flutter/core/constants/app_colors.dart';
+import 'package:ams_flutter/core/constants/app_icons.dart';
+import 'package:ams_flutter/core/constants/app_images.dart';
+import 'package:ams_flutter/core/constants/app_text_styles.dart';
+import 'package:ams_flutter/core/constants/dummy_models.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/bottom_navbar.dart';
+import 'package:ams_flutter/features/onboarding/presentation/widgets/text_field_widget.dart';
+import 'package:flutter/material.dart';
+import '../../../../core/constants/app_string.dart';
+import '../../../../core/constants/dummy_models.dart';
+import '../widgets/studentlist.dart';
+
+class MarkAttendance extends StatefulWidget {
+  @override
+  State<MarkAttendance> createState() => _MarkAttendanceState();
+}
+
+class _MarkAttendanceState extends State<MarkAttendance> {
+  late Size mq;
+
+  List<Map<String, dynamic>> dummyStudents = DummyModels.dummyStudents;
+  @override
+  Widget build(BuildContext context) {
+    mq = MediaQuery.of(context).size;
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(20.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            _buildHeader(context),
+            SizedBox(height: 16.0),
+            _buildDropDown(context),
+            SizedBox(height: 4),
+            TextFieldController(
+                textEditingController: TextEditingController(),
+                hinttext: "SearchStudent",
+                textInputType: TextInputType.name,
+                prefixIcon: Icon(Icons.search),
+                borderRadius: 40),
+                // SizedBox(height: 4.0),
+            Expanded(
+              child: SingleChildScrollView(
+                child: StudentList(
+                  students: dummyStudents
+                      .map((student) => Student(
+                            name: student['name'],
+                            rollno: student['rollno'],
+                            photoUrl: student['photoUrl'],
+                          ))
+                      .toList(),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      bottomNavigationBar: BottomNavBar(),
+    );
+  }
+  Widget _buildHeader(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Image.asset(
+          AppImages.arrow_image,
```

Comment:
Instead of using these Images You can use icons... these will be pixelated on bigger screens.